auto-ipsec/libreswan/README.md

```diff
@@ -1,36 +1,36 @@
 # Automatic IPsec Configuration with Libreswan
 
-Using the certificate distribution mechanism built into keylime, you can automatically configure keylime to use IPsec to securely communicate with other hosts in a network.  
+Using the certificate distribution mechanism built into keylime, you can automatically configure keylime to use IPsec to securely communicate with other machines in a network.  
 
 ## Overview
 
-These scripts allow keylime agents to create a virtual private network between them that is protected by cryptographic keys bootstrapped by keylime.  Keylime will also automatically remove failed agents from the network.  The scripts work by leveraging two features of keylime: automatic certificate generation/delivery, and the ability run scripts provided in the payload after successful bootstrapping.  The following high level actions will happen:
+These scripts allow machines running keylime agent to create a virtual private network between them that is protected by cryptographic keys bootstrapped by keylime.  Keylime will also automatically remove failed machines from the network.  The scripts work by leveraging two features of keylime: automatic certificate generation/delivery, and the ability run scripts provided in the payload after successful bootstrapping.  The following high level actions will happen:
 
-* the tenant will first run generate.py in this folder to generate the IPsec configuration files to pass to the tenant.  the tenant must specify the subnets/ips that they wish for agents to always use IPsec to communicate.
-* the tenant will generate a new CA (if one hasn't been generated yet) and then generate a new cert/key combo for the agent to be bootstrapped
-* the tenant will include the IPsec config files generated above with the `--include` option
-* tenant and verifier will bootstrap the agent and derive a key as normal
-* once the bootstrap key has been derived, keylime will decrypt and extract the zipped payload that includes both the cert/key and the ipsec config.
+* The tenant will first run generate.py in this folder to generate the IPsec configuration files to pass to the tenant. The tenant must specify the subnets/ips that they wish for agents to always use IPsec to communicate.
```

Comment:
It looks like this was changed for racoon but not libreswan: 

`they wish for agents to always use IPsec to communicate`

Should be wish for **machines** to always use